estatesdb-dev/00-namespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: estatesdb-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "Estates"
+    cloud-platform.justice.gov.uk/application: "Estates Database"
+    cloud-platform.justice.gov.uk/owner: "Programmes and Portfolios: matthew.stainsby@digital.justice.gov.uk"
```

Comment:
This needs to be a group email address for your team, rather than a personal email address. This is so that the organisation can keep track of namespaces and their purpose, even after the person who created it has left the organisation.

I'll approve this for now, but please set up a google group with a suitable email address for your production namespace.